db/migrate/20210407101956_add_responsibility_to_calculated_handover_dates.rb

```diff
@@ -0,0 +1,10 @@
+class AddResponsibilityToCalculatedHandoverDates < ActiveRecord::Migration[6.0]
+  def change
+    change_table :calculated_handover_dates do |t|
+      # This ought to be non-nullable, but we have existing data
+      t.boolean :pom_responsible
+      # This is deliberately nullable as the COM is sometimes not involved
+      t.boolean :com_responsible
```

Comment:
See my previous comments.

It'd be really nice if we could be consistent with these fields. I think they would be better suited as representing the entire `pom_responsibility` and `com_responsibility` `HandoverData` objects, rather than just one field (`responsible?`).

I don't like the feel of these special comments floating around continually needing to explain the different between the POM/COM responsibilities (i.e. one can be true/false, the other can be true/false/nil)